runtime/frequency/src/lib.rs

```diff
@@ -337,6 +337,28 @@ impl Contains<RuntimeCall> for PasskeyCallFilter {
 	}
 }
 
+pub struct MsaCallFilter;
+use pallet_frequency_tx_payment::types::GetAddKeyData;
+impl GetAddKeyData<RuntimeCall, AccountId, MessageSourceId> for MsaCallFilter {
+	fn get_add_key_data(call: &RuntimeCall) -> Option<(AccountId, AccountId, MessageSourceId)> {
+		match call {
+			#[cfg(feature = "runtime-benchmarks")]
+			RuntimeCall::System(frame_system::Call::remark { .. }) => None,
+
```

Comment:
curious, why do we need this if the default arm of the match returns `None` anyway?

Reply:
Good catch.  This originally returned something else, and during a debugging session I set it to None then forgot about it.  I think it's fine just to delete it but let me double check the benchmarks.